docs/source/getting-started.mdx

```diff
@@ -40,39 +35,48 @@ Your project directory now contains a `package.json` file.
 
 ## Step 2: Install dependencies
 Applications that run Apollo Server require two top-level dependencies:
-<!-- TODO: CHECK THIS LINK once packaged published -->
 * [`@apollo/server`](https://www.npmjs.com/package/@apollo/server) is the core library for Apollo Server itself, which helps you define the shape of your data and how to fetch it.
 * [`graphql`](https://npm.im/graphql) is the library used to build a GraphQL schema and execute queries against it.
 
-Run the following command to install both of these dependencies and save them in
+We'll also install the [`graphql-tag`](https://www.npmjs.com/package/graphql-tag) package, which includes a template literal tag that parses GraphQL query strings into the standard GraphQL AST.
```

Comment:
If we want to keep this as simple as possible, `graphql-tag` isn't necessary at all. We can provide `typeDefs` as a string to AS.